src/simulator/pysimtest/launch/dynamics/dynamics.py

```diff
@@ -0,0 +1,346 @@
+#!/usr/bin/env python
+
+# Basic ROS imports
```

Comment:
Does UWSim not have a physics model? What was the reasoning behind this?

Reply:
The entire dynamics.py file came in the UWsim installation, I did not write/edit any of it. It is meant to be a script that can be launched alongside UWsim. I believe another team is working on creating a custom version of this script tailored to our robot, but currently dynamics.py is not used in any way in the library. I will remove it to avoid confusion and wait until the custom model is built.